WebService/v1/Models/SimulationApiModel/SimulationStatistics.cs

```diff
@@ -0,0 +1,71 @@
+// Copyright (c) Microsoft. All rights reserved.
+
+using System;
+using Newtonsoft.Json;
+
+namespace Microsoft.Azure.IoTSolutions.DeviceSimulation.WebService.v1.Models.SimulationApiModel
+{
+    public class SimulationStatistics
+    {
+        [JsonProperty(PropertyName = "TotalMessagesSent")]
+        public long TotalMessagesSent { get; set; }
+
+        [JsonProperty(PropertyName = "AverageMessagesPerSecond")]
+        public double AverageMessagesPerSecond { get; set; }
+
+        [JsonProperty(PropertyName = "FailedMessagesCount")]
+        public long FailedMessagesCount { get; set; }
+
+        [JsonProperty(PropertyName = "ActiveDevicesCount")]
+        public long ActiveDevicesCount { get; set; }
```

Comment:
could you comments on all the fields, helping to understand what to expect? For instance what do we mean by "Active" ?